docs/isdubad-config.md

```diff
@@ -109,6 +109,14 @@ Valid values for `tlps` are the [Traffic Light Protocol](https://en.wikipedia.or
 - `feed_importer`: Name of the user that is doing the feed imports. Defaults to `feedimporter`.
 - `publishers_tlps`: Rules what the feed import is allowed to import. Defaults to `{ "*" = [ "WHITE", "GREEN", "AMBER", "RED" ] }`
 - `default_message`: The message that should be displayed inside the source manager.
+- `aes_key`: A crypto token in form of 64 character long hex string used to encrypt security sensitive\
+   fields in the database like passphrasses and private keys. Defaults to "".\
+   If an empty token is given, the server will generate a new one every time it starts.\
```

Comment:
Leaving this field empty generates a new token each time, which means that the saved passphrases will not be accessible anymore and lost. What about stopping the server if the field is empty instead to prevent a later error situation which is hard to understand?

Reply:
> Leaving this field empty generates a new token each time, which means that the saved passphrases will not be accessible anymore and lost. What about stopping the server if the field is empty instead to prevent a later error situation which is hard to understand?

I have considered this and I am still not sure if this good idea or not as not all potential users of ISDuBA are going to use the secured stuff.